grr/server/grr_response_server/gui/api_integration_tests/osquery_test.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+# Lint as: python3
+"""Integration tests for the Osquery flow, its api client and api endpoints."""
```

Comment:
If you capitalize "osquery" then I think "API" also should be written in uppercase.